app/components/mark_complete_component/view.html.erb

```diff
@@ -1,4 +1,4 @@
 <%= form_with model: @mark_complete_form, url: @path, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
-  <%= f.govuk_collection_radio_buttons :mark_complete, @mark_complete_options, :value, :name, legend: { size: 'm' } %>
+  <%= render :partial => 'forms/mark_complete/form', locals: { form: f, hint: @hint, legend: @legend} %>
```

Comment:
This doesn't feel right. A component should be isolated or at least consume another component but not rely on partials which are buried in another folders. 

Could we not maybe use same component but pass in a flag to either include or exclude the `<form>` elements?

Reply:
Good suggestion - had a bit of difficulty with this when mocking the form in the tests, but should be sorted now